sdk/core/core-lro/test/getYieldedValue.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "vitest";
```

Comment:
Was this moved from somewhere or is it a new helper? I think I remember `getYieldedValue` from a long time ago but can't find it now

Reply:
Oh I think I get it, we moved this and Matrix over from test-util to avoid taking a dependency on it for now. Seems fine for now :thumbsup: